web-animations-2/Overview.bs

```diff
@@ -2855,3 +2855,21 @@ Add:
 >     </div>
 >
 >     Note: Need to spec this properly somewhere.
+
+The <code>AnimationPlaybackEvent</code> interface {#the-animationplaybackevent-interface}
```

Comment:
nit: Can we put this before Model liveness (since this precedes Model liveness in https://drafts.csswg.org/web-animations-1/#the-animationplaybackevent-interface . Also I think this needs to be an h3 to be one of the sections under programming interface.

Reply:
done